app/Fission/Environment/IPFS/Types.hs

```diff
@@ -8,12 +8,14 @@ data Environment = Environment
   { url     :: !IPFS.URL     -- ^ IPFS client URL (may be remote)
   , timeout :: !IPFS.Timeout -- ^ IPFS timeout in seconds
   , binPath :: !IPFS.BinPath -- ^ Path to local IPFS binary
+  , gateway :: !IPFS.Gateway -- ^ Domain Name of IPFS Gateway
```

Comment:
How is this different from `url`?

Reply:
`url` is the url of the http interface for some node (ie `localhost:5001`)
`gateway` is the url of the gateway that we want to point dns records to (ie `ipfs.runfission.com`)

Reply:
Sorry, I'm clearly coming at this without enough context. What feature does this enable?

Reply:
actually doesn't enable a feature. This is fixing https://github.com/fission-suite/web-api/issues/162 (https://trello.com/c/I5ETfd7L/826-fix-gateway-in-env)

the issue was slightly misframed which I addressed in this comment: https://github.com/fission-suite/web-api/issues/162#issuecomment-556254891